cellprofiler_426.def

```diff
@@ -0,0 +1,55 @@
+Bootstrap:docker
+# use linux-amd64 ubuntu jammy micromamba 1.5
+From:mambaorg/micromamba@sha256:e678de11f8a47f2811f051a1329a7910bfbe056a49c63152a761d83dec3e5c4e
```

Comment:
Doing it by hash because I started developing this on my arm64 macOS so this was the easiest way to avoid architecture issues.